SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -326,22 +291,31 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	bool bGenerateDefaultLaunchConfig;
 
 	/** Returns which runtime variant we should use. */
-	TEnumAsByte<ESpatialOSRuntimeVariant::Type> GetSpatialOSRuntimeVariant() const { return RuntimeVariant; }
+	TEnumAsByte<ESpatialOSRuntimeVariant::Type> GetSpatialOSRuntimeVariant() const { return ESpatialOSRuntimeVariant::Standard; }
 
-	/** Returns the version information for the currently set variant*/
+	/** Returns the version information for the currently set runtime variant*/
 	const FRuntimeVariantVersion& GetSelectedRuntimeVariantVersion() const
 	{
-		return const_cast<USpatialGDKEditorSettings*>(this)->GetRuntimeVariantVersion(RuntimeVariant);
+		return const_cast<USpatialGDKEditorSettings*>(this)->GetRuntimeVariantVersion(ESpatialOSRuntimeVariant::Standard);
 	}
 
-	UPROPERTY(EditAnywhere, config, Category = "Runtime")
-	TEnumAsByte<ESpatialOSRuntimeVariant::Type> RuntimeVariant;
-
-	UPROPERTY(EditAnywhere, config, Category = "Runtime", AdvancedDisplay)
+	UPROPERTY(EditAnywhere, config, Category = "Runtime", meta = (DisplayName = "Runtime Versions"))
 	FRuntimeVariantVersion StandardRuntimeVersion;
 
-	UPROPERTY(EditAnywhere, config, Category = "Runtime", AdvancedDisplay)
-	FRuntimeVariantVersion CompatibilityModeRuntimeVersion;
+	/** Whether to use the GDK-associated SpatialOS inspector version for local deployments, or to use the one specified in the
```

Comment:
just to quickly check: did we have a tech writer look at the new tooltips?

Reply:
I will double check with tech writers, cheers.

Reply:
Tech writers have done a pass!